packages/keychain/src/components/Fees.tsx

```diff
@@ -27,6 +27,7 @@ export function Fees({
         maxFee.overall_fee,
         18,
         token.price,
+        true, // Enable exact precision for fees
```

Comment:
I think we want to modify packages/keychain/src/components/starterpack/total-cost.tsx